jsonnet/thanos-mixin/dashboards/store.libsonnet

```diff
@@ -0,0 +1,281 @@
+local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'store.json':
```

Comment:
Let's move the gRPC RED dashboards to the top as that's how others interface with Thanos Store.